tests/Kiota.Builder.Tests/Extensions/OpenApiUrlTreeNodeExtensionsTests.cs

```diff
@@ -1072,6 +1072,20 @@ public void repro4085()
         Assert.Equal("\\path\\{differentThingId-id}", differentThingId.Path);
         Assert.Equal("{+baseurl}/path/{differentThingId%2Did}", differentThingId.GetUrlTemplate());
     }
+
+    [Theory]
+    [InlineData("{path}", "WithPath")]
+    [InlineData("archived{path}", "archivedWithPath")]
```

Comment:
isn't this going to be a breaking change for Microsoft Graph SDKs (functions)? could we trigger a smoke generation to double check?

Reply:
There's no diff in the graph metadata (see diff between kiota main and this branch at https://github.com/microsoftgraph/msgraph-sdk-dotnet/compare/kiota/v1.0/pipelinebuild/144499..kiota/v1.0/pipelinebuild/144497)
This is because 
- indexers in graph metadata do not have characters outside the indexer name like `archived{path}` but will just be {path}
- Functions will always be generated with the parameter name and `=` separating the placeholder character.

Reply:
Thanks for confirming, resolving.